docs/src/start/search.md

```diff
@@ -0,0 +1,258 @@
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+Sample 65536 out of 1000000 to train kmeans of 8 dim, 256 clusters
+```
+
+!!! warning "Index Creation Performance"
+    If you're trying this on your own data, make sure your vector (dimensions / num_sub_vectors) % 8 == 0, or else index creation will take much longer than expected due to SIMD misalignment.
+
+## Vector Search with the ANN Index
+
+You can now perform the same search operation using your newly created index and see the dramatic performance improvement.
+
+```python
+sift1m = lance.dataset(uri)
+
+import time
+
+tot = 0
+for q in samples:
+    start = time.time()
+    tbl = sift1m.to_table(nearest={"column": "vector", "q": q, "k": 10})
+    end = time.time()
+    tot += (end - start)
+
+print(f"Avg(sec): {tot / len(samples)}")
+print(tbl.to_pandas())
+```
+
+Expected output:
+```
+Avg(sec): 0.0009334301948547364
+       id                                             vector         score
+0  378825  [20.0, 8.0, 121.0, 98.0, 37.0, 77.0, 9.0, 18.0...  16560.197266
+1  143787  [11.0, 24.0, 122.0, 122.0, 53.0, 4.0, 0.0, 3.0...  61714.941406
+2  356895  [0.0, 14.0, 67.0, 122.0, 83.0, 23.0, 1.0, 0.0,...  64147.218750
+3  535431  [9.0, 22.0, 118.0, 118.0, 4.0, 5.0, 4.0, 4.0, ...  69092.593750
+4  308778  [1.0, 7.0, 48.0, 123.0, 73.0, 36.0, 8.0, 4.0, ...  69131.812500
+5  222477  [14.0, 73.0, 39.0, 4.0, 16.0, 94.0, 19.0, 8.0,...  69244.195312
+6  672558  [2.0, 1.0, 0.0, 11.0, 36.0, 23.0, 7.0, 10.0, 0...  70264.828125
+7  365538  [54.0, 43.0, 97.0, 59.0, 34.0, 17.0, 10.0, 15....  70273.710938
+8  659787  [10.0, 9.0, 23.0, 121.0, 38.0, 26.0, 38.0, 9.0...  70374.703125
+9  603930  [32.0, 32.0, 122.0, 122.0, 70.0, 4.0, 15.0, 12...  70583.375000
+```
+
+!!! note "Performance Note"
+    Your actual numbers will vary by your storage. These numbers are from local disk on an M2 MacBook Air. If you're querying S3 directly, HDD, or network drives, performance will be slower.
+
+## Tune the Search Parameters
+
+You need to adjust search parameters to balance between speed and accuracy, finding the optimal settings for your use case.
+
+The latency vs recall is tunable via:
+- **nprobes**: how many IVF partitions to search
```

Comment:
this is out of date, now there is min and max nprobes

Reply:
https://github.com/danielfrg/mkdocs-jupyter

Reply:
(just to be clear this will be a lot of effort, we don't need to do it in this PR, but want to align the direction)